public/coffee/ide/editor/ShareJsDoc.coffee

```diff
@@ -46,6 +46,7 @@ define [
 			@_doc.on "change", () =>
 				@trigger "change"
 			@_doc.on "acknowledge", () =>
+				@lastAcked = new Date() # note time of last ack from server
```

Comment:
Is this trigger on receiving a collaborator update? A comment to this effect might be useful, either way.